angularjs/angular.d.ts

```diff
@@ -746,6 +747,18 @@ declare namespace angular {
         <T>(array: T[], expression: string|((value: T) => any)|(((value: T) => any)|string)[], reverse?: boolean): T[];
     }
 
+    interface IFilterLinky {
+        /**
+         * Finds links in text input and turns them into html links. Supports http/https/ftp/mailto and plain email address links.
+         * @param text Input text.
+         * @param target Window (_blank|_self|_parent|_top) or named frame to open links in.
+         * @param attributes Add custom attributes to the link element.
+         * @return Html-linkified and sanitized text.
+         * see https://docs.angularjs.org/api/ngSanitize/filter/linky
+         */
+        (text: string, target: string, attributes:Object|Function): string;
```

Comment:
Done. I've made only the `attributes` parameter optional as only it stays as optional in official docs: https://docs.angularjs.org/api/ngSanitize/filter/linky